### PR TITLE
Liked page and alert

### PR DESCRIPTION
**Pull request by tarunsankhla**
Ignore comments and console
review
LikePage
LikeContext
Toast
VideoCard

### CODE REVIEW

File: src/components/UI/Cards/LikedVideoCards/LikedVideoCards.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useLikes } from '../../../../context/LikesContext';
+import { VAR_ENCODE_TOKEN } from '../../../../utils/Route';
```
Comment (rahulyadav139) [inline src/components/UI/Cards/LikedVideoCards/LikedVideoCards.jsx:5]: Great you are storing the route path in a variable. This is new to me. Great work.

File: src/components/UI/Cards/PlayListVideoCards/PlayListVideoCards.jsx [hunk 2]
```diff
@@ -32,7 +36,8 @@ function PlayListVideoCards({ props, setPlayListVideos }) {
             })()
         }
         catch (err) {
-            console.log(err)
+            console.log(err);
```
Comment (rahulyadav139) [inline src/components/UI/Cards/PlayListVideoCards/PlayListVideoCards.jsx]: remove unnecessary console.log.

File: src/pages/SignUpPage/SignupPage.jsx [hunk 3]
```diff
@@ -102,29 +103,37 @@ function SignupPage() {
...
+            console.log(VerifyCredentials())
+           
+                console.log(object)
+                var res = await axios.post("/api/auth/signup", object);
+                console.log(res);
+                if (res.status === 201) {
+                    var token = res?.data?.encodedToken;
+                    localStorage.setItem(VAR_ENCODE_TOKEN, token)
+                    var user = res?.data?.createdUser;
+                    var userId = res?.data?.createdUser?._id;
```
Comment (rahulyadav139) [inline src/pages/SignUpPage/SignupPage.jsx]: I am not sure but do you need actually three-level optional chaining.

**Review by rahulyadav139 — APPROVED** [review]

File: src/pages/LikedPage/LikePage.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  const { likesContextArray, setLikesContextArray } = useLikes();
+  useEffect(() => {
+		try {
+			(async () => {
+				var res = await axios.get("/api/user/likes", {
+					headers: {
+						authorization: localStorage.getItem(VAR_ENCODE_TOKEN),
+					},
+				});
+				console.log(res);
```
Comment (deekshithmd) [inline src/pages/LikedPage/LikePage.jsx]: Suggestion: You can remove these console.log if not required...

File: src/components/UI/Cards/VideoCards/VideoCards.css [hunk 5]
```diff
@@ -84,6 +84,21 @@ a{
 	width: 100%;
 }
 
+.video-badge-container{
+	height: fit-content !important;
```
Comment (deekshithmd) [inline src/components/UI/Cards/VideoCards/VideoCards.css:88]: suggestion: align codes and check whether the usage of !important is necessary or not

File: src/pages/SignUpPage/SignupPage.jsx (hunk 3 above)
Comment (tarunsankhla) [reply src/pages/SignUpPage/SignupPage.jsx]: If there is issue within backend and it doesn't give data or created User so here it shouldn't fail and handle

File: src/components/UI/Cards/VideoCards/VideoCards.css (hunk 5 above)
Comment (tarunsankhla) [reply src/components/UI/Cards/VideoCards/VideoCards.css:88]: I actually did that but the hierarchy of structure is not getting me result for this so need to do inportant

File: src/pages/LikedPage/LikePage.jsx (hunk 4 above)
Comment (tarunsankhla) [reply src/pages/LikedPage/LikePage.jsx]: done

File: src/components/UI/Cards/PlayListVideoCards/PlayListVideoCards.jsx (hunk 2 above)
Comment (tarunsankhla) [reply src/components/UI/Cards/PlayListVideoCards/PlayListVideoCards.jsx]: done

**Review by PuneetP16 — APPROVED** [review]
PR changes looks good to merge.

**Pull request merged**